File: server/src/Bananagrams.Dal/Models/GameAnagramType.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Bananagrams.Dal.Models;

[Table("game_anagram_types")]
public class GameAnagramType
{
    [Key]
    [Column("id")]
    public int Id { get; set; }
    [Column("max_attempts")]
    public int MaxAttempts { get; set; }
    [Column("title")]
    public string Title { get; set; }
    [Column("time_allowed")]
    public int TimeAllowed { get; set; }
}

File: server.java/bananagrams/src/main/java/coe/bananagrams/entity/GameAnagramType.java
package coe.bananagrams.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.*;

@Getter
@Setter
@Entity
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "game_anagram_types")
public class GameAnagramType {
    @Id
    @Column(name = "id", updatable = false, nullable = false)
    private int id;
    @Column(name ="max_attempts", nullable = false)
    private int maxAttempts;
    @Column(name = "title", nullable = false)
    private String title;
    @Column(name = "time_allowed", nullable = false)
    private int timeAllowed;
}
